packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -85,12 +85,12 @@
   /// @type Value
   /// @access public
   /// @group button
-  $button-padding: 0.875rem 63px 0.875rem 15px !default !global;
+  $button-padding: calc(0.875rem - 3px) 60px calc(0.875rem - 3px) 12px !default !global;
```

Comment:
Wanted to double-check the change here is still intentional.

Reply:
yes this change is intentional!